examples/amass_human_poses/config.yaml

```diff
@@ -0,0 +1,88 @@
+# Args: <path_to_shape-net-core> <output_dir>
+{
+  "version": 3,
+  "setup": {
+    "blender_install_path": "/home_local/<env:USER>/blender/",
+    "pip": [
+      "h5py", "git+https://github.com/abahnasy/smplx", "git+https://github.com/abahnasy/human_body_prior"
+    ]
+  },
+  "modules": [
+    {
+      "module": "main.Initializer",
+      "config":{
+        "global": {
+          "output_dir": "<args:1>"
+        }
+      }
+    },
+    {
+      "module": "loader.AMASSLoader",
+      "config": {
+        "data_path": "<args:0>",
+        "sub_dataset_id": "CMU",
+        "body_model_gender": "male",
+        "subject_id": "10",
+        "sequence_id": "1",
+        "frame_id": "600",
+        "cf_set_shading": "SMOOTH"
```

Comment:
Can that be done by default by the `AMASSLoader` I can not imagine that someone would want a flat shading.

Reply:
Now, it is handled from inside the module